models/mpas_atm/model_mod.f90

```diff
@@ -3051,7 +3051,7 @@ subroutine write_model_time_restart(ncid, dart_time)
 type(time_type),     intent(in) :: dart_time
 
 integer :: year, month, day, hour, minute, second
-character(len=64) :: timestring
+character(len=64) :: timestring = ''
```

Comment:
Note to check before we merge this. Initializing in the declaration is an implicit save in Fortran, so `timestring` is only initialed on the first call to `write_model_time_restart`. We may want to change the initialization to be separate:

```suggestion
character(len=64) :: timestring
timestring = ''
```